skyvern/webeye/scraper/domUtils.js

```diff
@@ -749,6 +750,11 @@ function buildTreeFromBody() {
         processElement(child, elementObj.id);
       });
       return elementObj;
+    } else if (element.tagName.toLowerCase() === 'iframe') {
+      let iframeElementObject = buildElementObject(element, true);
+
+      elements.push(iframeElementObject);
+      resultArray.push(iframeElementObject);
```

Comment:
Should we skip the iframes here? 

```
function buildElementObject(element, interactable) {
```
Interactable here means that LLM can take an action such as CLICK/INPUT_TEXT/SELECT_OPTION etc on this specific element. Is that possible for this root iframe element here? Or are we making it possible via building an element tree for each of the iframes?

Reply:
The intention of including iframes in the elements list was to set unique id on the iframe for mapping later on and build element tree for each iframe. Not sure how to implement that without building the element object.